apps/dashboard/components/app-users/smart-tables/app-users-table.tsx

```
@@ -32,170 +18,178 @@ const ITEMS_PER_PAGE = 100;
+    switch (key) {
+      case 'identity':
+        return `${users.length} users`;
+      case 'talentProtocolCheckmark':
+        const verifiedCount = users.reduce((sum, user) => {
+          if ('personaData' in user) {
+            // For ParsedUser, we can access personaData directly
+            const persona = user.personaData;
+            return sum + (persona?.identities.talentProtocol?.profile.human_checkmark ? 1 : 0);
+          }
+          return sum;
+        }, 0);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)